src/tests/Common/testenvironment.proj

```diff
@@ -174,6 +174,14 @@
     <TestEnvironment Include="jitstress2_jitstressregs0x10" JitStress="2" JitStressRegs="0x10" />
     <TestEnvironment Include="jitstress2_jitstressregs0x80" JitStress="2" JitStressRegs="0x80" />
     <TestEnvironment Include="jitstress2_jitstressregs0x1000" JitStress="2" JitStressRegs="0x1000" />
+    <!-- There are multiple cases of `jitstress_random_*` so a single pipeline can run random JitStress multiple times without fear
+         of a problem due to repeating the same scenario name. Note that this file is invoked with only one of these at a time.
+    -->
+    <TestEnvironment Include="jitstress_random_1" JitStress="random" />
+    <TestEnvironment Include="jitstress_random_2" JitStress="random" />
+    <TestEnvironment Include="jitstress_random_3" JitStress="random" />
```

Comment:
I am not sure if I followed why in `yml` you just have `jitstress_random_1` and `jitstress_random_2` while here there are 3,4 and 5?

Reply:
Was just "future-proofing" it but they're currently not used. I could remove them.

Reply:
Can you please remove the unneeded environments?

Reply:
There's kind of an open question: should the pipeline run one random job, or two, or three, or... ? Running more than one gives more coverage and shares the build time, but also (obviously) takes more resources. I'm doing 2 now. Not sure what the right number is.

Reply:
Sure, so may be we should delete `random_3` onward entries.